tests/view/test_generic_views.py

```diff
@@ -0,0 +1,15 @@
+from timeless.views import ListView
+from timeless.customers.models import Customer
+
+
+class TestListView(ListView):
+    model = Customer
+
+
+def test_view_ordering(app):
+    view = TestListView()
+    query = Customer.query
+    with app.test_request_context("/?ordering=id,-first_name"):
+        query = view.sort_query(query)
+        assert "ORDER BY customers.id ASC, customers.first_name DESC" \
+               in str(query)
```

Comment:
@vladarefiev Can you add other a few test cases besides the Happy Flow? Seeing the method implementations, I ave a few questions: 

* what happens if the params are not properly sepparated by ``,``?
* what happens if the attribute does not actually exist in the model? 
* what happens if someone specifies a ``+``?
* other negative cases...

My problem is that the sort_query method is not really protected against malformed input.

Reply:
@amihaiemil agree, will add a puzzle for validation query pararms